src/Uno.UI.RuntimeTests/Helpers/ViewModelBase.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Runtime.CompilerServices;
+
+namespace Uno.UI.RuntimeTests.Helpers
+{
+	public class ViewModelBase : INotifyPropertyChanged
+	{
+		public event PropertyChangedEventHandler PropertyChanged;
+
+		protected void RaiseAndSetIfChanged<T>(ref T backingField, T value, [CallerMemberName] string propertyName = null)
```

Comment:
```suggestion
		protected void SetAndRaiseIfChanged<T>(ref T backingField, T value, [CallerMemberName] string propertyName = null)
```

:)

Reply:
the name is actually taken from reactiveui's ReactiveObject.RaiseAndSetIfChanged, or inspired from
but SetAndRaiseIfChanged is okay too